backend/src/panels/YeomanUIPanel.ts

```diff
@@ -0,0 +1,124 @@
+import * as _ from 'lodash';
+import * as path from 'path';
+import * as os from "os";
+import * as vscode from 'vscode';
+import { YeomanUI } from "../yeomanui";
+import { RpcExtension } from '@sap-devx/webview-rpc/out.ext/rpc-extension';
+import { YouiLog } from "../youi-log";
+import { GeneratorFilter } from '../filter';
+import backendMessages from "../messages";
+import { OutputChannelLog } from '../output-channel-log';
+import { YouiEvents } from "../youi-events";
+import { VSCodeYouiEvents } from '../vscode-youi-events';
+import Environment = require('yeoman-environment');
+import { AbstractWebviewPanel } from './AbstractWebviewPanel';
+
+
+export class YeomanUIPanel extends AbstractWebviewPanel {
+	public static YEOMAN_UI = "Yeoman UI";
+	private static channel: vscode.OutputChannel;
+
+	public loadYeomanUI(uiOptions?: any) {
+		this.disposeWebviewPanel();
+		const webViewPanel = this.createWebviewPanel();
+		this.setWebviewPanel(webViewPanel, uiOptions);
+	}
+
+	public toggleOutput() {
+		this.outputChannel.showOutput();
+	}
```

Comment:
toggleOutput command only from yeoma-ui webview ?!

Reply:
Yes, they did not defined it in the explore gens mockup

Reply:
They might forgot it.